src/gosling-track/gosling-track.ts

```diff
@@ -2,14 +2,14 @@ import * as PIXI from 'pixi.js';
 import PubSub from 'pubsub-js';
 import * as uuid from 'uuid';
 import { isEqual, sampleSize, uniqBy } from 'lodash-es';
-import { scaleLinear } from 'd3-scale';
+import { ScaleLinear } from 'd3-scale';
```

Comment:
I think use `import type` again, for consistency (for maybe all the new imports in this file?) I think all are types.